public/app/features/teams/CreateTeam.test.tsx

```diff
@@ -3,13 +3,23 @@ import userEvent from '@testing-library/user-event';
 import React from 'react';
 
 import { BackendSrv, setBackendSrv } from '@grafana/runtime';
+import * as hooks from 'app/core/components/RolePicker/hooks';
```

Comment:
```suggestion
import { useRoleOptions } from 'app/core/components/RolePicker/hooks';
```
Wildcard imports aren't the best for readability, plus in this case it's only one hook, so can be imported explicitly.

Reply:
Changed, thanks!